main-project/src/boards/boards.controller.ts

```diff
@@ -0,0 +1,35 @@
+import { Body, Controller, Get, Logger, Param, Post } from '@nestjs/common';
+import { BoardsService } from './boards.service';
+import { CreateBoardDto } from './dto/create-board.dto';
+import { Board } from './entity/board.entity';
+
+@Controller('boards')
+export class BoardsController {
+  private logger = new Logger('BoardsController');
+  constructor(private boardService: BoardsService) {}
+
+  @Get()
+  getAllBoards(): Promise<Board[]> {
+    this.logger.debug(`Get all boards.`);
+
+    return this.boardService.getAllBoards();
```

Comment:
```suggestion
    const boards: Board[] = this.boardService.getAllBoards();
    const response = {
        success: true,
        data: boards,
    };
    
    return response;
```

바로 리턴해주는 것도 좋지만 협업에서는 프론트에서 응답 결과를 알아볼 수 있게 성공 여부를 포함해서 보내주는 것도 좋습니다.
'상태 코드로 판단하면 되지 않냐' 라는 의문이 들 수 있는데 현업에서 모든 리턴값은 200 혹은 400으로 해준다는 점 참고해주세요. 다만 현업은 현업이고 프로젝트 진행할 때는 상태코드 모두 응답에 맞도록 설정해주는 버릇을 들이면 좋습니다~

암튼 결론은 프론트에서 응답의 성공 유무정도는 직관적으로 알 수 있게 success를 보내주는 게 좋습니다~

아래 컨트롤러 return 모두 해당!

Reply:
추가로 응답에 대한 타입도 지정해서 넣어주면 좋아요~~

Reply:
응답에 대한 타입도 지정해서 넣을게요 감사합니다!!